README.md

```diff
@@ -191,6 +191,22 @@ foo.push(1);
 foo.push(2);
 ```
 
+*Question: What are the values of `num`, `obj1.item` and `obj2.item`?*
+```javascript
+function change(num, obj1, obj2)
+{
+  num = num * 10;
+  obj1.item = "changed";
+  obj2 = {item: "changed"};
+}
+
+var num = 10;
+var obj1 = {item: "unchanged"};
+var obj2 = {item: "unchanged"};
+
+change(num, obj1, obj2);
+```
```

Comment:
Can you try to make this code snippet consistent with the rest of the code examples? Spacing, single quotes, `{` shouldn't be on it's own line, ...